<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerClass</class>
 <widget class="QMainWindow" name="ViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Viewer/img/main_logo.ico</normaloff>:/Viewer/img/main_logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLineEdit" name="stockIineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkPushButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stockExchangeComboBox">
        <item>
         <property name="text">
          <string>上交所</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>深交所</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="currencyComboBox">
        <item>
         <property name="text">
          <string>人民币</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>美元</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="exchangeTypeComboBox">
        <item>
         <property name="text">
          <string>股票</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>期货</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="openingPriceLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>开盘</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="openingPriceValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="amountLabel">
        <property name="text">
         <string>数量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="amountValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minLabel">
        <property name="text">
         <string>最低</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="risingStopPriceLabel">
        <property name="text">
         <string>涨停价</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="risingStopPriceValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="currentLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>最新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="turnoverLabel">
        <property name="text">
         <string>交易量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="turnoverValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxLabel">
        <property name="text">
         <string>最高</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="droppingStopPriceLabel">
        <property name="text">
         <string>跌停价</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="droppingStopPriceValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="buyPrice">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>竞买价</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxBuyLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sellPrice">
        <property name="text">
         <string>竞卖价</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minSellLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="buyTableWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="sellTableWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="timeLineLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="text">
         <string>分时线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dayKLineLabel">
        <property name="text">
         <string>日k线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>signal1()</signal>
 </slots>
</ui>
